### DISCUSSION TITLE
Tad 0.14.0

### DISCUSSION THREAD

**Discussion by antonycourtney** [antonycourtney/tad#270] [Announcements]
## What's Changed
* Updated to DuckDb 1.0
* Export filtered tables as Parquet (as well as CSV)
* Fix bug with formatting numbers in scientific notation.
* Other minor bug fixes and dependent package upgrades

**Full Changelog**: https://github.com/antonycourtney/tad/compare/v0.13.0...v0.14.0

<hr /><em>This discussion was created from the release <a href='https://github.com/antonycourtney/tad/releases/tag/v0.14.0'>Tad 0.14.0</a>.</em>

**Comment by victoriano** [comment]
I’m not sure if it’s just me, but in this version, it takes a full minute for Tad to open. I switched back to version 0.12, which opens any dataset with 50K rows almost instantly. I’m using a MacBook M1.
